### PR TITLE
Remove unused deps

### PR DESCRIPTION
**Pull request by mmso**
Remove dependencies used in projects

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,25 +36,13 @@
...
-    "eslint-plugin-flowtype": "^5.1.3",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.1.0",
     "execa": "^4.0.2",
     "fast-sass-loader": "^1.5.0",
     "favicons-webpack-plugin": "^3.0.1",
     "file-loader": "^6.0.0",
-    "file-saver": "^2.0.2",
```
Comment (dhoko) [inline package.json:57]: we don't need it anymore ?

File: package.json [hunk 2]
```diff
@@ -40,25 +36,13 @@
...
     "html-webpack-plugin": "^4.0.4",
     "imagemin-mozjpeg": "^9.0.0",
     "imagemin-webpack-plugin": "^2.4.0",
-    "json-loader": "^0.5.7",
```
Comment (dhoko) [inline package.json:61]: idem?

File: package.json (hunk 1 above)
Comment (mmso) [reply package.json:57]: yes we need it but it's depended on from the project that uses it instead

File: package.json (hunk 2 above)
Comment (mmso) [reply package.json:61]: doesn't seem like it's being used

File: package.json (hunk 1 above)
Comment (dhoko) [reply package.json:57]: what do you mean?

Comment (dhoko) [reply package.json:57]: it's not a webpack loader/plugin :facepalm:

**Pull request merged**